update_cache.sh

```diff
@@ -64,6 +64,10 @@ for arg in "$@"; do
 		clone_if_not_present WikibaseEdtf.git https://github.com/ProfessionalWiki/WikibaseEdtf.git
 		fetch_all WikibaseEdtf.git
 
+		# AutomatedValues
+		clone_if_not_present AutomatedValues.git https://github.com/QuentinJanuel/AutomatedValues
```

Comment:
Strongly recommend not using the (already out-of-date) fork. Here is the main repo: https://github.com/ProfessionalWiki/AutomatedValues